<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>installWindow</class>
 <widget class="QMainWindow" name="installWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>thinkerAI Installer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>インストール先</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="install_dir_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="install_dir_select">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>インストールログ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="install_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>インストール</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="install_log_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
